ivre/zgrabout.py

```diff
@@ -236,6 +243,29 @@ def zgrap_parser_http(data, hostrec):
             })
             banner += (b"Server: " + utils.nmap_decode_data(server[0]) +
                        b"\r\n\r\n")
+        if headers.get('www_authenticate'):
+            auths = headers.get('www_authenticate')
+            for auth in auths:
+                if auth[:4].lower() == 'ntlm' and auth[5:]:
+                    msg = utils.decode_b64(auth[4:].strip().encode())
+                    msg = utils.ntlm_extract_info(msg)[5:]
+                    infos = {k: (int(v) if k == 'ntlm-version' else
```

Comment:
I don't know if it is useful to use a first function (`utils.ntlm_extract_info()`) that produces base64-encoded content that you now have do decode. Maybe you could have the function in `utils` return a `dict` object and use that object either directly or encode its data?